osdk-core/hal/src/dji_log.cpp

```diff
@@ -100,15 +102,18 @@ Log::print()
 Log&
 Log::print(const char* fmt, ...)
 {
-  if ((!release) && vaild)
+  // If print() method was called from nonPrintableInstance
+  // it means that this log type is disabled
+  if (this != &nonPrintableInstance)
   {
     va_list args;
     va_start(args, fmt);
     mutex->lock();
     vprintf(fmt, args);
+    fflush(stdout);
```

Comment:
Force unwritten data to be write to stdout. Without this, log messages are sometimes delayed